_layouts/post.html

```diff
@@ -38,7 +38,9 @@
         <hr/>
         <p class="cross-post"><a href="{{ page.canonical.href }}">This post was cross-posted to {{ page.canonical.name }}.</a></p>
         {% endif %}
+      <a class="btn-hire btn-u-orange ls-s1" href="/careers/craftsman/">We are hiring!</a>

```

Comment:
Links are inline elements. Should it be in a `<p>`, `<div>` or similar so we don't have a mix of block and inline elements in the containing `<div>`?